cfgov/mega_menu/jinja2tags.py

```diff
@@ -0,0 +1,57 @@
+from django.conf import settings
+from django.template import loader
+from django.utils.translation import get_language_from_request
+
+from flags.state import flag_enabled
+from jinja2 import Markup, contextfunction
+from jinja2.ext import Extension
+
+from mega_menu.frontend_conversion import FrontendConverter
+from mega_menu.models import Menu
+
+
+def mega_menu_template(context):
+    request = context.get('request')
+
+    if flag_enabled('MEGA_MENU_VAR_1', request=request):
+        return '_includes/organisms/mega-menu-var-1.html'
+    elif flag_enabled('MEGA_MENU_VAR_2', request=request):
+        return '_includes/organisms/mega-menu-var-2.html'
+    else:
+        return '_includes/organisms/mega-menu.html'
+
+
+def select_menu_for_context(context):
+    language = context.get('language')
+
+    if not language:
+        request = context['request']
+        language = get_language_from_request(request, check_path=True)
+
+    try:
+        return Menu.objects.get(language=language[:2])
+    except Menu.DoesNotExist:
+        return Menu.objects.get(language=settings.LANGUAGE_CODE[:2])
```

Comment:
It looks like this would error if there's no English menu, although I'm not sure that's a use case we need to support?

Reply:
:+1: I figured this was a reasonable thing to assume, that'd there always be at least one menu defined corresponding to the default language. I've added a new test in 874b2058d that verifies that the code raises `Menu.DoesNotExist` if there isn't a default menu.